Makefile

```diff
@@ -46,6 +47,12 @@ stop:
 	docker-compose down && \
 	popd
 
+reset:
+	pushd ${radish34} && \
+	docker-compose down && \
+	docker volume rm radish34_mongo-buyer radish34_mongo-supplier1 radish34_mongo-supplier2 radish34_mongo-merkle-tree-volume radish34_chaindata && \
```

Comment:
I think `make reset` is more aptly named for this.

Can you take a look at `make clean` while you are here and make sure it isn't too aggressive? Meaning... doing things that should be done here in `reset`.

I think `clean` is still a valid target but it should just wipe all the node modules.

Reply:
`make clean` might be a little aggressive in the sense that it will prune ALL docker containers and networks, not just ones related to this `baseline/radish34` project. In order to limit the pruning to this project I think we would have to explicitly list our container names and network names

Reply:
For sure. I think we should put all docker-related stuff in `reset` and it should be less aggressive.

And then `make clean` just delegates to npm clean...